backend/src/lib/transformer/index.spec.js

```diff
@@ -23,10 +23,15 @@ describe('Transformer', () => {
   describe('playback.getCurrentTrack', () => {
     const data = 'data'
 
-    it('does the right thing', () => {
+    it('tranforms when we have data', () => {
       Transformer('mopidy::playback.getCurrentTrack', data)
       expect(TransformTrack).toHaveBeenCalledWith(data)
     })
+
+    it('does not tranform when we have no data', () => {
+      Transformer('mopidy::playback.getCurrentTrack', null)
+      expect(TransformTrack).not.toHaveBeenCalledWith()
```

Comment:
Smallest review point possible: as it's `not` here I think it can just be `not.toHaveBeenCalled()`

Reply:
well spotted, that's a typo from when copy and pasting a previous expectation.